flow360/component/flow360_params/flow360_params.py

```diff
@@ -580,6 +581,166 @@ def validate_boundary(cls, values):
+        An instance of the component class ReferenceFrame.
+
+    Example
+    -------
+    >>> rf = ReferenceFrame(
+            center=(0, 0, 0),
+            axis=(0, 0, 1),
+            omega_radians=1
+        )
+    """
+
+    theta_radians: Optional[str] = pd.Field(alias="thetaRadians")
+    theta_degrees: Optional[str] = pd.Field(alias="thetaDegrees")
+    omega_radians: Optional[float] = pd.Field(alias="omegaRadians")
+    omega_degrees: Optional[float] = pd.Field(alias="omegaDegrees")
+    center: Coordinate = pd.Field(alias="centerOfRotation")
+    axis: Axis = pd.Field(alias="axisOfRotation")
+    parent_volume_name: Optional[str] = pd.Field(alias="parentVolumeName")
+    is_dynamic: Optional[bool] = pd.Field(alias="isDynamic")
+
+    # pylint: disable=missing-class-docstring,too-few-public-methods
+    class Config(Flow360BaseModel.Config):
+        require_one_of = [
+            "omega_radians",
+            "omega_degrees",
+            "theta_radians",
+            "theta_degrees",
+            "is_dynamic",
+        ]
+
+
+class FluidDynamicsVolumeZone(VolumeZoneType):
+    """FluidDynamicsVolumeZone type"""
+
+    model_type = pd.Field("FluidDynamics", const=True)
+    reference_frame: Optional[ReferenceFrame] = pd.Field(alias="ReferenceFrame")
+
+
+class _GenericVolumeZonesWrapper(Flow360BaseModel):
+    v: Union[FluidDynamicsVolumeZone, HeatTransferVolumeZone]
+
+
+class VolumeZones(Flow360SortableBaseModel):
+    """:class:`VolumeZones` class for setting up volume zones
+
+    Parameters
+    ----------
+    <zone_name> : Union[FluidDynamicsVolumeZone, HeatTransferVolumeZone]
+
+    Returns
+    -------
+    :class:`VolumeZones`
+        An instance of the component class VolumeZones.
+
+    Example
+    -------
+    >>> zones = VolumeZones(
+            zone1=FluidDynamicsVolumeZone(),
+            zone2=HeatTransferVolumeZone(thermal_conductivity=1)
+        )
+    """
+
+    @pd.root_validator(pre=True)
+    def validate_zone(cls, values):
+        """Validator for zone list section
+
+        Raises
+        ------
+        ValidationError
+            When zone is incorrect
+        """
+        return _self_named_property_validator(
+            values, _GenericVolumeZonesWrapper, msg="is not any of supported volume zone types."
+        )
+
+
+class AeroacousticOutput(Flow360BaseModel):
+    """:class:`AeroacousticOutput` class for configuring output data about acoustic pressure signals
+
+    Parameters
+    ----------
+    observers : List[Coordinate]
+        List of observer locations at which time history of acoustic pressure signal is stored in aeroacoustic output
+        file. The observer locations can be outside the simulation domain, but cannot be inside the solid surfaces of
+        the simulation domain.
+
+    Returns
+    -------
+    :class:`AeroacousticOutput`
+        An instance of the component class AeroacousticOutput.
+
+    Example
+    -------
+    >>> aeroacoustics = AeroacousticOutput()
```

Comment:
lets give example that means something, for example `observers=[(0, 0, 0), (0, 0, 1)]`

Reply:
Fixed in a01150b, also added minimal entries for optional parameters in the documentation